dao/src/main/java/com/epam/esm/criteria/certificates/AllGiftCertificateCriteria.java

```diff
@@ -0,0 +1,34 @@
+package com.epam.esm.criteria.certificates;
+
+import com.epam.esm.criteria.Criteria;
+import com.epam.esm.entity.GiftCertificate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+/**
+ * This Criteria gets all GiftCertificate entities in DB.
+ *
+ * @since 1.0
+ */
+@Component("allGCCriteria")
+public class AllGiftCertificateCriteria extends AbstractGiftCertificateCriteria implements Criteria<GiftCertificate> {
+
+    private final static String GET_ALL_QUERY = "SELECT * FROM " + TABLE_NAME;
```

Comment:
It's not a good practice to use * in your select query

Reply:
commit 7c617a9